docs/pages/ds-display_L3_Low-Level_TestSpecification.md

```diff
@@ -0,0 +1,185 @@
+# Device Settings Display L3 Low Level Test Specification and Procedure Documentation
```

Comment:
Can you move all Display document to dsDisplay folder

Reply:
Done